vector/src/main/res/layout/fragment_qr_code_scanner.xml

```diff
@@ -21,38 +21,50 @@
     </com.google.android.material.appbar.AppBarLayout>
 
     <me.dm7.barcodescanner.zxing.ZXingScannerView
-        android:id="@+id/scannerView"
+        android:id="@+id/qrScannerView"
         android:layout_width="0dp"
         android:layout_height="0dp"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/appBarLayout" />
 
-    <!-- TODO In the future we could add a toggle to switch the flash, and other possible settings -->
+    <Button
+        android:id="@+id/userCodeMyCodeButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/layout_vertical_margin_big"
+        android:maxWidth="160dp"
+        android:text="@string/user_code_my_code"
+        android:visibility="gone"
```

Comment:
Thanks, that's better to be gone by default to avoid blinking on screen.
But can you add `tools:visibility="visible"` for a better layout preview?